src/explore-education-statistics-common/src/services/publicationService.ts

```diff
@@ -53,6 +53,14 @@ export interface PublicationSummary {
   summary: string;
 }
 
+export interface BasicPublicationContact {
+  id?: string;
```

Comment:
Should this just be merged with `PublicationContact`? It's a bit odd that this can be optional too. Would have thought they'd all have the `id`

Reply:
It's a bit awkward because of the slightly different types for the admin and front end contacts, the frontend one doesn't have the id on. But yeh can probably be merged.

Reply:
I'd probably only merge them if `teamName` isn't optional. Just noticed that it looks like it is for some reason?

Reply:
Yeah looks like it is, though I don't know why. Will leave them separate.